app/scripts/components/common/cookie-consent/index.scss

```diff
@@ -0,0 +1,25 @@
+#cookie-consent{
```

Comment:
Have you had a chance to familiarize yourself with utility functions (such as https://designsystem.digital.gov/utilities/display/, https://designsystem.digital.gov/design-tokens/z-index/ ) in USWDS? I think a lot of this custom style can be replaced with the USWDS utility class - and this will also give us some coherence on the units.

Reply:
I did, but wasn't familiar with all of the available tokens. I will get those adjusted.